knime_extension/src/nodes/spatialstatistics.py

```diff
@@ -56,6 +56,29 @@ def _var_col_exists_or_preset(
     )
 
 
+def get_id_col_parameter(
```

Comment:
Instead of copying the code from the spatialmodels file we could move the existing method into the modelin_utils file and expose the since_version as parameter with 1.1.0 as default value